### PR TITLE
Tutorial Update

### PR DESCRIPTION
**Pull request by MrLeap**
Added Anchors.
Added the hello world source. 
Added a few screenshots. 
Fixed some other misc issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Getting started with Node.js
```
Comment (jcollard) [inline README.md:21]: I would like a table of contents linking to each of the headers.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-[Atom](https://atom.io/)
-[Visual Studio Code](https://code.visualstudio.com/)
-[Notepad++](https://notepad-plus-plus.org/) (Windows Only)
+- [Atom](https://atom.io/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Notepad++](https://notepad-plus-plus.org/) (Windows Only)
 
-## Installation
+## [installation](installation)
```
Comment (jcollard) [inline README.md]: Capitalize

**Review by jcollard — CHANGES_REQUESTED** [review]
Looking great! Thanks :)

File: README.md (hunk 1 above)
Comment (MrLeap) [reply README.md:21]: I amended the pr.

**Comment by jcollard** [comment]
LGTM! Thanks.

**Pull request merged**